lxd/db/cluster/update.go

```diff
@@ -74,6 +74,13 @@ var updates = map[int]schema.Update{
 	35: updateFromV34,
 	36: updateFromV35,
 	37: updateFromV36,
+	38: updateFromV37,
+}
+
+// Attempt to add missing project features.networks feature to default project.
+func updateFromV37(tx *sql.Tx) error {
+	tx.Exec("INSERT INTO projects_config (project_id, key, value) VALUES (1, 'features.networks', 'true');")
```

Comment:
I'm wondering if it would be safer to:

1) check if a project named `"default"` exists
2) use the ID of that project as value for the `project_id` column in the insert query

The reason is that there might not be any default project at all, or it might have a different ID than `1` since it was deleted and recreated.

Reply:
@freeekanayaka makes sense, will do.

Reply:
@freeekanayaka have updated.